folio-spring-i18n/src/main/java/org/folio/spring/config/TranslationConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.spring.config;
+
+import java.util.Locale;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * An {@link org.springframework.context.annotation.Configuration Configuration} bean to hold the
+ * location of the current translation directory.  By default, this is set from the application
+ * property {@code folio.translation=directory} in your {@code application.properties} or
+ * {@code application.yaml} file.  Note that the proceeding and trailing slashes are required.
+ */
+@Getter
+@Configuration
+public class TranslationConfiguration {
+
+  private String translationDirectory;
+
+  private Locale defaultLocale;
+
+  @Autowired
+  public TranslationConfiguration(
+    @Value(
+      "${folio.translation-directory:/translations/}"
+    ) String translationDirectory,
+    @Value("#{T(java.util.Locale).getDefault()}") Locale defaultLocale
```

Comment:
There's additional fallback to `en`, too. I added the server default locale because absolutely defaulting to english felt a bit wrong, and to cover cases where perhaps non-community modules with only one non-english locale are created, which I could certainly see occurring if a non-english institution made there own module.

If you'd prefer, I can remove this; just figured additional fallbacks would be better.

Reply:
done